### PR TITLE
Document 'default_group_priority' and 'supervisor' priority

### PR DESCRIPTION
**Pull request by Adimus11**
## 🚀 Links

- [Feature branch](https://developers.labs.livechat.com/docs/feature/API-8995-add-info-about-supervisor-priority)
- [Jira](https://livechatinc.atlassian.net/browse/XXX-XXX)

## ✅ Checklist (for API docs)

- Changelog
- API versions
- Web & RTM
- **Table:** Available methods/webhooks/pushes
- **Table:** Specifics

For more guidelines, see [Updating API docs](https://livechatinc.atlassian.net/wiki/spaces/PAT/pages/761529400/Updating+API+docs).

### Content

- New content
- Proofreading/content fixes
  
### Features (for the website)

- New feature
- Bug fix
- Breaking change

### CODE REVIEW

File: content/management/changelog/index.mdx [hunk 1]
```diff
@@ -21,6 +21,10 @@ The developer preview version provides a preview of the upcoming changes to the
 
 </Warning>
 
+### Agents
+- Documented `default_group_priority` for agents.
```
Comment (lpmichalik) [inline content/management/changelog/index.mdx:25]: you reverted that part :)

**Review by lpmichalik — APPROVED** [review]

File: content/management/changelog/index.mdx [hunk 2]
```diff
@@ -21,6 +21,9 @@ The developer preview version provides a preview of the upcoming changes to the
 
 </Warning>
 
+### Agents
+- Added support for `supervisor` priority for group priorities.
```
Comment (oliwiapolec) [inline content/management/changelog/index.mdx:25]:
```suggestion
- Added support for the `supervisor` priority for the group priority parameter.
```

File: content/management/configuration-api/index.mdx [hunk 3]
```diff
@@ -94,6 +94,7 @@ __*__ Possible values for the `groups[].priority` parameter:
 | `first`        |                                                           The highest chat routing priority. Agents with the `first` priority get chats before others from the same group, e.g. Bots can get chats before regular Agents.                                                           |
 | `normal`       | The medium chat routing priority. Agents with the `normal` priority get chats before those with the `last` priority, when there are no Agents with the `first` priority available with free slots in the group. Corresponds to the **Primary** priority option in the LiveChat App. |
 | `last`         |              The lowest chat routing priority. Agents with the `last` priority get chats when there are no Agents with the `first` or `normal` priority available with free slots in the group. Corresponds to the **Secondary** priority option in the LiveChat App.               |
+| `supervisor`   |                                                                                                         Agent works as `supervisor` so it will not be assigned to any chats.                                                                                                        |
```
Comment (oliwiapolec) [inline content/management/configuration-api/index.mdx:97]:
```suggestion
| `supervisor`   |                                                                                                         Agents with the `supervisor` priority will not get any chats assigned automatically.                                                                                                        |
```

File: content/management/configuration-api/index.mdx [hunk 4]
```diff
@@ -358,6 +359,7 @@ __*__ Possible values for the `groups[].priority` parameter:
...
```
Comment (oliwiapolec) [inline content/management/configuration-api/index.mdx:362]:
```suggestion
| `supervisor`   |                                                                                                         Agents with the `supervisor` priority will not get any chats assigned automatically.                                                                                                        |
```

File: content/management/configuration-api/index.mdx [hunk 5]
```diff
@@ -1063,11 +1065,12 @@ Creates a new bot. To make the bot available for receiving chats or webhooks, us
...
+| --------------- | :-------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------: |
+| `first`         |                         The highest chat routing priority. Agents with the `first` priority get chats before others from the same group, e.g. Bots can get chats before regular Agents.                         |
+| `normal`        | The medium chat routing priority. Agents with the `normal` priority get chats before those with the `last` priority, when there are no Agents with the `first` priority available with free slots in the group. |
+| `last`          |               The lowest chat routing priority. Agents with the `last` priority get chats when there are no Agents with the `first` or `normal` priority available with free slots in the group.                |
+| `supervisor`    |                                                                       Bot works as `supervisor` so it will not be assigned to any chats.                                                                        |
```
Comment (oliwiapolec) [inline content/management/configuration-api/index.mdx:1073]: As I see we were consistent with the possible values being the same as for Agents for the Bot-related values as well 🤔  I _think_ we should stick to that and keep:
```suggestion
| `supervisor`    |                                                                       Agents with the `supervisor` priority will not get any chats assigned automatically.                                                                        |
```
but I will need to consult this with Ola and let you know :)

File: content/management/configuration-api/index.mdx [hunk 6]
```diff
@@ -1091,6 +1094,8 @@ __****__ The table below presents the possible values for the `default_group_pri
 | `first`         |                         The highest chat routing priority. Agents with the `first` priority get chats before others from the same group, e.g. Bots can get chats before regular Agents.                         |
 | `normal`        | The medium chat routing priority. Agents with the `normal` priority get chats before those with the `last` priority, when there are no Agents with the `first` priority available with free slots in the group. |
 | `last`          |               The lowest chat routing priority. Agents with the `last` priority get chats when there are no Agents with the `first` or `normal` priority available with free slots in the group.                |
+| `supervisor`    |                                                                       Bot works as `supervisor` so it will not be assigned to any chats.                                                                        |
```
Comment (oliwiapolec) [inline content/management/configuration-api/index.mdx:1097]: the same as for the above - will check with Ola

File: content/management/configuration-api/index.mdx [hunk 7]
```diff
@@ -1196,11 +1201,12 @@ Updates an existing Bot.
...
+| --------------- | :-------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------: |
+| `first`         |                         The highest chat routing priority. Agents with the `first` priority get chats before others from the same group, e.g. Bots can get chats before regular Agents.                         |
+| `normal`        | The medium chat routing priority. Agents with the `normal` priority get chats before those with the `last` priority, when there are no Agents with the `first` priority available with free slots in the group. |
+| `last`          |               The lowest chat routing priority. Agents with the `last` priority get chats when there are no Agents with the `first` or `normal` priority available with free slots in the group.                |
+| `supervisor`    |                                                                       Bot works as `supervisor` so it will not be assigned to any chats.                                                                        |
```
Comment (oliwiapolec) [inline content/management/configuration-api/index.mdx:1209]: and same here :)

**Comment by oliwiapolec** [comment]
a quick additional question: I noticed that for the Bot's possible values you've removed the "corresponds with" infos, e.g. `Corresponds to the **Bot** priority option in the LiveChat App`
but left this note for the Agent's values. Is there any specific reason for that? :) I personally think that's a nice addition to these notes as when creating a Bot other than for the group General you can switch the bot's prio in AA 🤔

**Comment by Adimus11** [comment]
@oliwiapolec AFAIK there is no possibility to set `supervisor` via Agent App, so it's corresponding to nothing :c

**Review by oliwiapolec — APPROVED** [review]

File: content/management/changelog/index.mdx [hunk 8]
```diff
@@ -21,6 +21,9 @@ The developer preview version provides a preview of the upcoming changes to the
 
 </Warning>
 
+### Agents
```
[the remark this answers is not in the document]
Comment (litwinow) [reply content/management/changelog/index.mdx:24]: yup, thx

**Review by ale-kacperczyk — APPROVED** [review]
🙌🏻

**Pull request merged**